apps/mainsite/views.py

```diff
@@ -56,6 +62,64 @@ def error404(request, *args, **kwargs):
     }))
 
 
+@login_required
+def frontend_redirect(request):
+    return redirect('http://badgr-ui.local.overhang.io:4000/')
+
+
+def authenticate_lms_user(request):
+    token = request.headers.get('Authorization')
+
+    if not token:
+        return False
+    
+    try:
+        decoded_data = jwt.decode(token, options={"verify_signature": False})
+        email = decoded_data.get('email')
+        if not email:
+            return 
+
+        url = "http://local.overhang.io:8000/api/badges/v1//verify-lms-token/"
```

Comment:
Take LMS Host from the settings or we can also add a setting i.e LMS_JWT_VERIFICATION_URL.